planning/Discovery/work-items.md

```diff
@@ -0,0 +1,51 @@
+# Web of Things (WoT) Discovery
+## Work Items
+Prioritized list of work items related to Discovery.
+- Detailed descriptions of work items will be in a single list (in a separate document) to accomodate
+  work items that impact more than one deliverable.
+- Each work item has a link to its detailed description, and a link to the requirement(s) it satisfies or helps satisfy.
+- Note that requirements will also link in turn to the use cases that motivate it.
+- Priority is given by the ordering in the list, from most to least important.
+     - No ties allowed; this is called force-ranking and is intentional.
+ 
+### To do:
+- Assign "categories" to work items to cluster them into a smaller number of "feature sets".
+    - Right now have categories for "Direct" (internal Discovery feature),
+      "Dependency" (involved in more than one deliverable), and "Process" (not really a spec feature,
+      but a procedural matter).
+- Following not yet actually prioritized, we may need to split into smaller work items first
+- Update base URL to work-item list after it gets moved
+- Need to add requirements and links from each work item, below only a couple of examples are given
+     - Link to requirements in wot-architecture
+     - We could also put these in a document, or in MD files under each deliverable
+     - However, a single list of requirements would allow us to address requirements that span more than one deliverable
+
+### Prioritized List
+1. Direct - [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
+    * Needs to be broken into smaller work-items to better support prioritization.
+2. Direct - [Discovery of Thing Models](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-thing-models-workitem) 
+3. Direct - [Discovery CoAP Directory](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-coap-dir-workitem)
+4. Direct - [Discovery JSON Path Query Language](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-jsonpath-query-language-workitem)
```

Comment:
This is more important in my opinion, it is a core functionality that makes the Directory useful in more scenarios, what is the difference with query filters though?

Reply:
The problem with all the "official" query languages is that the specs are not final.  "Query filters" would be simple, fixed-function query mechanisms that would not have a dependency on a query language.  It's also a way for us to do geolocation filters without depending on extending the official query languages (there is a GeoSPARQL, but it's even more complicated to implement than SPARQL...).